### DISCUSSION TITLE
'copilot is encountering temporary issues' happens again - again

### DISCUSSION THREAD

**Discussion by cblackuk** [community/community#31731] [Copilot Conversations]
Hi,

Is anybody else having this issue? It started this morning on 3 different PCs simultaneously.

I tried the usual reinstalls, VSCode upgrades, the lot - nothing works ;-(

Help please.
![image](https://user-images.githubusercontent.com/28607803/188122489-43c5ab40-7d26-4ed4-8f93-85c76c78e896.png)

Could not comment here, hence new thread -> https://github.com/orgs/community/discussions/27983

```powershell
curl --verbose https://copilot-proxy.githubusercontent.com/_ping
*   Trying 20.80.64.28:443...
* Connected to copilot-proxy.githubusercontent.com (20.80.64.28) port 443 (#0)
* schannel: disabled automatic use of client certificate
* ALPN: offers http/1.1
* ALPN: server accepted http/1.1
> GET /_ping HTTP/1.1
> Host: copilot-proxy.githubusercontent.com
> User-Agent: curl/7.83.1
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Content-Security-Policy: default-src 'none'; sandbox
< Content-Type: application/json
< Strict-Transport-Security: max-age=31536000
< Date: Fri, 02 Sep 2022 10:40:49 GMT
< Content-Length: 33
<
{"now":1662115249,"status":"ok"}
* Connection #0 to host copilot-proxy.githubusercontent.com left intact
```

**Comment by davecheney** [comment]
@cblackuk i'm sorry you are having difficulties. It looks like the clock on your computer is not keeping time well and copilot is constantly trying to re-login. Please correct your clock.

**Reply by cblackuk** [reply]
What do you mean? :-D @davecheney I have it running on 3 different PCs, in 2 (I believe) different time zones. All appear to be set correctly to me :-)

Help? :-)

**Reply by cblackuk** [reply]
Is there any way we can have a call or Teams chat please? :-) @davecheney  I really need it to work :-) please help :-)

**Reply by cblackuk** [reply]
@davecheney any ideas, please? :-)

**Comment by jasonkneen** [comment]
I had the same issue -- nothing wrong with my clock. I've installed the nightly extension and re-logged in and no errors currently

**Reply by cblackuk** [reply]
I just tried:
![image](https://user-images.githubusercontent.com/28607803/188205973-d6329e89-3ab3-4477-b9a2-162ddce931a5.png)

Still not working ;-(

**Reply by cblackuk** [reply]
![image](https://user-images.githubusercontent.com/28607803/188206178-592ac2b5-8bb4-4792-8bd4-cefd512fdc8c.png)

**Reply by cblackuk** [reply]
[2022-09-02 12:30:03.270] [exthost] [info] ExtensionService#_doActivateExtension GitHub.copilot, startup: false, activationEvent: 'onStartupFinished'

**Comment by cblackuk** [comment]
As if by magic, it works now on all 3 PCs and I did literally nothing to fix it. If somebody did - thank you :-)

**Comment by jasonkneen** [comment]
I had it again today -- removed extension, restarted, installed nightly etc. Nothing and still had the issue.

Then went into settings and removed:

```
"github.copilot.advanced": {
        "debug.overrideEngine": "copilot-logit",
  },
```

and works now.

**Reply by DeltaDesigns** [reply]
> I had it again today -- removed extension, restarted, installed nightly etc. Nothing and still had the issue.
> 
> Then went into settings and removed:
> 
> ```
> "github.copilot.advanced": {
>         "debug.overrideEngine": "copilot-logit",
>   },
> ```
> 
> and works now.

Worked for me as well, been having that problem for a couple days now and uninstalled/reinstalled like you did
